app/services/foreman_rh_cloud/cloud_connector.rb

```diff
@@ -12,11 +12,20 @@ def install
       target_host = foreman_host
       composer = nil
 
+      input = {
+        :satellite_cloud_connector_user => service_user.login,
+        :satellite_cloud_connector_password => token_value,
+      }
+
+      if (http_proxy = Setting[:http_proxy])
```

Comment:
You should use [`ForemanRhCloud.proxy_setting(logger: logger)`](https://github.com/theforeman/foreman_rh_cloud/blob/2519e2fff3dd08b4e87663cb9a465ed98d6856f8/lib/foreman_rh_cloud.rb#L40) to conform with proxy selection logic for the rest of rh_cloud plugin.